Assets/Scripts/PlayerController/Player.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class Player : MonoBehaviour
+{
+    string PlayerName;
```

Comment:
why?

Reply:
Different characters need different names

Reply:
shoot logic should not affect player and i don't see any need in this. Let's add it when it necessary?